### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by cmaneu**
Hello @SnehaGunda and @shsagir,
As discussed, here is my proposal for an updated readme 😉. 

## Folder structure

On top of the readme, I've moved the samples in the "e2e" folder in a new "feature samples" folder. These samples are not "End to end" samples, and putting them unorganized in a folder with that name is misleading. Hence this proposal. My team is working on "real" end-to-end samples that could end-up in the "e2e-samples" in the near future. 

If we move on with this structure, we need to ping initial authors (there is at least one blog post to update).

### CODE REVIEW

**Comment by cmaneu** [comment]
Hello @SnehaGunda and @shsagir,
Could you have a look at this PR?

File: README.md [hunk 1]
```diff
@@ -1,33 +1,62 @@
...
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+## What's Inside
+
+This repository is organized into different sections to help you navigate through various samples and resources easily.
+
+### Documentation Samples
+
+- **Location:** `docs-samples` folder.
+- **Description:** These samples support the Microsoft Fabric official documentation and Microsoft Learn modules. They include both code examples and sample data.
```
Comment (SnehaGunda) [inline README.md:14]:
```suggestion
- **Description:** These samples are referenced within the Microsoft Fabric official documentation and Learn training content. They include both code examples and sample data.
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,62 @@
...
+
+### Feature Samples
+
+- **Location:** `feature-samples` folder.
+- **Description:** Focus on specific features of Microsoft Fabric, showcasing how to leverage them in your projects.
```
Comment (SnehaGunda) [inline README.md:19]:
```suggestion
- **Description:** These samples focus on specific features of Microsoft Fabric, demonstrating how to leverage them in your projects.
```

File: README.md [hunk 3]
```diff
@@ -1,33 +1,62 @@
...
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+While we're currently not accepting contributions from external contributors, we're eager to hear about the samples you'd like to share. If you have a sample proposal:
+
+1. Ensure any datasets used are licensed appropriately for sharing.
+2. Open a draft pull request with the following
+    - A new folder under `community-samples` with a short name for your sample
+    - A `readme.md` sample describing the objective of the sample
+    - If the sample code is not yet written, please wait for confirmation in the PR disucssion before engaging time to create the sample.
```
Comment (SnehaGunda) [inline README.md:45]: Will there be some evaluation criteria/best practices before merging the PRs?

**Pull request merged**